packages/create-remix/templates/fly-stack/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# base node image
+FROM node:16-bullseye-slim as base
+
+# set for base and all layer that inherit from it
+ENV NODE_ENV production
+
+# Install openssl for Prisma
+RUN apt-get update && apt-get install -y openssl
+
+# Install all node_modules, including dev dependencies
+FROM base as deps
+
+WORKDIR /myapp
+
+ADD package.json package-lock.json ./
+RUN npm install --production=false
+
+# Setup production node_modules
+FROM base as production-deps
+
+WORKDIR /myapp
+
+COPY --from=deps /myapp/node_modules /myapp/node_modules
+ADD package.json package-lock.json ./
+RUN npm prune --production
+
+# Build the app
+FROM base as build
+
+WORKDIR /myapp
+
+COPY --from=deps /myapp/node_modules /myapp/node_modules
+
+ADD prisma .
+RUN npx prisma generate
+
+ADD . .
+RUN npm run prepare
```

Comment:
While looking at and trying out the PR I stumbled across this line. In order for this step to work, doesn't there have to be a `prepare` script in the `package.json`?

Reply:
ah yes! I meant `postinstall` 🤦

Reply:
thanks for catching that @philip-weber 🙏